### ISSUE TITLE
How to apply my neural network method to this work?

### ISSUE THREAD

**Issue by Jeremiah-HL** [vboussot/ImpactLoss#2]
In this rep, I wanna figure a way to compose my code(deep learning nn) with Impact loss. However, I didn't find out a way to make the loss work for because it is realized with Elastix and parametersMap file.

**Comment by vboussot** [comment]
Hi Jeremiah,

The repo was originally focused on Elastix, but we’ve just added a PyTorch version of the IMPACT loss.
You can now use it directly from [IMPACT.py](https://github.com/vboussot/ImpactLoss/issues/IMPACT.py) and there’s a short usage example at the end of the README.

Feel free to try it and open an issue if you run into any problems.
